src/constant/api-path.ts

```diff
@@ -0,0 +1,3 @@
+export const API_PATH = {
+  REFERSH_TOKEN: '/token',
```

Comment:
백엔드 개발자와 협의해서 api endpoint는 수정하는게 추후에 헷갈리지 않을 것 같기도 하네요~
오타도 확인부탁드립니다
```suggestion
  REFRESH_TOKEN: '/token',
```